### ISSUE TITLE
Simulation freezing while reading Plumed script

### ISSUE THREAD

**Issue by mdpoleto** [openmm/openmm-plumed#51]
Hi, 

I have been facing this problem for a while and I could not find a solution for it. I ran a 500ns metadynamics on my system using both OpenMM and Plumed, which was not enough to achieve convergence of my FES. I want to extend my simulation, but my simulation freezes while reading the Plumed script as shown on Production.log below.

[Production.log](https://github.com/openmm/openmm-plumed/files/6665478/Production.log)

My plumed script is intended to create a cylinder volume based metadynamics, as below.
[metady_cyl_K2.txt](https://github.com/openmm/openmm-plumed/files/6665501/metady_cyl_K2.txt)

My OpenMM script is this:
[Production_script.zip](https://github.com/openmm/openmm-plumed/files/6665511/Production_script.zip)

I have seen this freezing behavior in different systems and metadynamics protocols as well. I have faced this issue while running locally in my PC and in a HPC facility.

Any ideas?

**Comment by raimis** [comment]
Hi @mdpoleto,

Thank for reporting the issue.

- What does the "freezing behavior" means exactly? Does CPU doing some work or its idle?
- I don't see any error messages? Have you recorded `stderr` too?
- Could you include all the files (including the system files) necessary to reproduce the issue?
- Is it possible to reproduce the issue quickly without running a 500 ns simulation?

**Comment by mdpoleto** [comment]
Hi @raimis 

I followed your suggestion of checking the reproducibility of the issue in shorter simulations and I figured out the problem. In shorter simulations (and smaller systems), the time Plumed takes to read is shorter, but I could detect it. 

It turns out my Plumed script has the flag CALC_RCT on its METAD protocol and, while restarting my simulation, Plumed takes some time to re-read all deposited gaussians in order to continue its calculation of CALC_RCT. Since my system is somewhat big and it had ran for 500ns, it took ~20min to read everything. Afterwards, the simulation continued without any problem.

Thank you for you suggestions and I am sorry for the inconvenience.
